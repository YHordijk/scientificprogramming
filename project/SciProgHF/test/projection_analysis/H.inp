**DIRAC
.TITLE
Fragment H
.WAVE FUNCTION
.ANALYZE
**GENERAL
.ACMOUT
**HAMILTONIAN
.ONESYS
**WAVE FUNCTIONS
.SCF
*SCF
.CLOSED SHELL
2 0
**ANALYZE
.MULPOP
.PRIVEC
*PRIVEC
.AOLAB
*END OF
