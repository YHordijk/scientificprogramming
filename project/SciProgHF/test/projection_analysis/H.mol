WHATEVER
Fragment H
Base : cc-pVDZ
C   1    
        1.    1
H     0.00000000          0.00000000          0.00000000
LARGE BASIS cc-pVDZ
FINISH
