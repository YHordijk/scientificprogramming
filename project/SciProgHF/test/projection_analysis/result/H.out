Master : DIRAC allocating 50000000 words of memory
******************************************************************************
*                                                                            *
*                                O U T P U T                                 *
*                                   from                                     *
*                                                                            *
*                   @@@@@    @@   @@@@@     @@@@     @@@@@                   *
*                   @@  @@        @@  @@   @@  @@   @@                       *
*                   @@  @@   @@   @@@@@    @@@@@@   @@                       *
*                   @@  @@   @@   @@ @@    @@  @@   @@                       *
*                   @@@@@    @@   @@  @@   @@  @@    @@@@@                   *
*                                                                            *
*                                                                            *
%}ZS)S?$=$)]S?$%%>SS$%S$ZZ6cHHMHHHHHHHHMHHM&MHbHH6$L/:<S///</:|/:|:/::!:.::--:
$?S$$%$$$$?%?$(SSS$SSSHMMMMMMMMMMMMMMMMMM6H&6S&SH&&k?6$r~::://///::::::.:::-::
(%?)Z??$$$(S%$>$)S6HMMMMMMMMMMMMMMMMMMMMMMR6M]&&$6HR$&6(i::::::|i|:::::::-:-::
$S?$$)$?$%?))?S/]#MMMMMMMMMMMMMMMMMMMMMMMMMMHM1HRH9R&$$$|):?:/://|:/::/:/.::.:
SS$%%?$%((S)?Z[6MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM&HF$$&/)S?<~::!!:::::::/:-:|.
SS%%%%S$%%%$$MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMHHHHHHM>?/S/:/:::':/://:/::-::
?$SSSS?%SS$)MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM/4?:S:/:::/:::/:/:::.::
S$(S?S$%(?$HMMMMMMMMMMMMMMMMM#&7RH99MMMMMMMMMMMMMMMMMMHHHd$/:::::/::::::-//.:.
(?SS(%)S&HMMMMMMMMMMMMMMMMM#S|///???$9HHMMMMMMMMMDSZ&1S/?</>?~:///::|/!:/-:-:.
$S?%?<H(MMMMMMMMMMMMMMMMR?`. :.:'::<>:``?/*?##*)$:/>       `((%://::/:::::/::/
S$($$)HdMMMMMMMMMMMMMMMP: . `   `  `    `      `-            `Z<:>?::/:::::|:i
c%%%&HMMMMMMMMMMMMMMMM6:                                      `$%)>%%</>!:::::
S?%/MMMMMMMMMMMMMMMMMMH-                                        /ZSS>?:?~:;/::
$SZ?MMMMMMMMMMMMMMMMMH?.                                        "&((/?//?|::::
$%$%&MMMMMMMMMMMMMMMMM:.                                          ?%/S:</::/::
($?}&HMMMMMMMMMMMMMMMM>:                                          $%%<?/i:|i::
Z$($&MMMMMMMMMMMMMMMMHk(:.  . -                                   .S/??/!:/:|:
(??$<HMMMMMMMMMMMMMMMFk|:   -.-.                                  :%/%/(:/:ii|
SZ(S?]MMMMMMMMMMMMMMHS?:- -  ::.:                                  |/S:</::?||
$%)$$(MMMMMMMMMMMMMMR):`:. :.:::`,,/bcokb/_                       :S?%?|~:/:/:
%$$%$)[[?$?MMMMMMMMMM: :.:-.::::$7?<&7&MMMMMMM#/           _ .. ..:</?:(:/::::
$$$?Z?HHH~|/MMMMMMMMM/`.-.:.:/:%%%%?dHMMMMMMMMMMH?,-   .,bMMMM6//./i~/~:<:::/:
4$($S$M//::S?ZHMMMMMH/:.':::.:/%S/&MMHMMMMMMMMRM&><   ,HMMMMMMMF  :::?:///:|::
)[$S$S($|_i:#>::*H&?/::.::/:"://:?>>':&HMHSMMMM$:'-   MMHMMMMHHT .)i/?////:::/
$$[$$>$}:dHH&$$--?S::-:.:::--/-:``./::>%Zi?)&/?`:.'   `H?$T*" `  /%?>%:)://ii|
$&=&/ZS}$RF<:?/-.|%r/:::/:/:'.-.-..|::S//!`"``          >??:    `SS<S:)!/////|
Z&]>b[Z(Z?&%:::../S$$:>:::i`.`. '-.'  `                         ,>%%%:>/>/!|:/
$$&/F&1$c$?>:>?/,>?$$ZS/::/:-: ...                              |S?S)S?<~:::::
&$&$&$k&>>|?<:<?((/$[/?)i~/:/. - -                              S?:%%%?/:::/::
=[/Z[[Fp[MMH$>?Z&S$$$/$S///||..-           -.-                  /((S$:%<:///:/
$&>1MHHMMMM6M9MMMM$Z$}$S%/:::.'.            .:/,,,dcb>/:.       ((SSSS%:)!//i|
MMMMMMMMMMMR&&RRRHR&&($(?:|i::-             .:%&S&$[&H&''     ../>%;/?>??:<::>
MMMMMMMMMMMMS/}S$&&H&[$SS//:::.:.   . . .v</Jq1&&D]&M&<,      :/::/?%%)S>?://:
MMMMMMMMMMMM?}$/$$kMM&&$(%/?//:..'.  .|//1d/`://?*/*/"` '     .:/(SS$%(S%)):%~
MMMMMMMMMMMM(}$$>&&MMHR#$S%%:?::.:|-.':;&&b/D/$p=qpv//b/~'   :/~~%%??$=$)Z$S+;
MMMMMMMMMMMM[|S$$Z1]MMMMD[$?$:>)/::: :/?:`'???bD&{b<<-'     .,:/)|SS(}Z/$$?/<S
MMMMMMMMMMMM||$)&7k&MMMMH9]$$??Z%|!/:i::'  `` .             SS?SS?Z/]1$/&$c/$S
MMMMMMMMMMMM| -?>[&]HMMMMMMMH1[/7SS(?:/..-` ::/Sc,/_,     _<$?SS%$S/&c&&$&>//<
MMMMMMMMMMMMR  `$&&&HMM9MMMMMMM&&c$%%:/:/:.:.:/??/''"    _MMHk/7S/]dq&1S<&&></
MMMMMMMMMMMMM?  :&96MHMMMMMMMMMMMHHk[S%(<<:// '         ,MMMMMMM&/Z6H]DkH]1$&&
MMMMMMMMMMMMMD    99H9HMMMMMMMMMMMMMMMb&%$<:i.:....    .MMMMMMMMM6HHHRH&H&H1SF
MMMMMMMMMMMMMM|   `?HMMMMMMMMMMMMMMMMMMMHk6k&>$&Z$/?_.bHMMMMMMMMMMM&6HRM9H6]Zk
MMMMMMMMMMMMMMM/    `TMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMHMH6RH&R6&
MMMMMMMMMMMMMMMM    -|?HMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMFHH6HMD&&
MMMMMMMMMMMMMMMMk  ..:~?9MMMMMMMMMMMMM#':MMMMMMMMMMMMMMMMMMMMMMMMMMMMM9MHkR6&F
MMMMMMMMMMMMMMMMM/  .-!:%$ZHMMMMMMMMMR' dMMMMMMMMMMMMMMMMMMMMMMMMMMMMM9MRMHH9&
MMMMMMMMMMMMMMMMMML,:.-|::/?&&MMMMMM' .MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMHRMH&&6
MMMMMMMMMMMMMMMMMMMc%>/:::i<:SMMMMMMHdMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMHHM&969k
MMMMMMMMMMMMMMMMMMMMSS/$$/(|HMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMHH&HH&
MMMMMMMMMMMMMMMMMMMM6S/?/MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMR96H1DR1
MMMMMMMMMMMMMMMMMMMMM&$MHMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMHMH691&&
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMH&R&9Z
MMMMMMMMMMMMMMMMMMMMMMMMMRHMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMH&96][6
MMMMMMMMMMMMMMMMMMMMMMMMp?:MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM96HH1][F
MMMMMMMMMMMMMMMMMMMMMMMM> -HMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMH&1k&$&
******************************************************************************
*                                                                            *
*         =======================================================            *
*                     Program for Atomic and Molecular                       *
*          Direct Iterative Relativistic Allelectron Calculations            *
*         =======================================================            *
*                                                                            *
*                        VERSION 04.1    SEPTEMBER 2004                      *
*                                                                            *
******************************************************************************
*                                                                            *
*    Written by:                                                             *
*                                                                            *
*    Hans Joergen Aa. Jensen  University of Southern Denmark    Denmark      *
*    Trond Saue               CNRS/ULP Strasbourg               France       *
*    Lucas Visscher           Vrije Universiteit Amsterdam      Netherlands  *
*                                                                            *
*    with contributions from:                                                *
*                                                                            *
*    Vebjoern Bakken          University of Oslo                Norway       *
*    Ephraim Eliav            University of Tel Aviv            Israel       *
*    Thomas Enevoldsen        University of Southern Denmark    Denmark      *
*    Timo Fleig               University of Duesseldorf         Germany      *
*    Olav Fossgaard           University of Tromsoe             Norway       *
*    Trygve Helgaker          University of Oslo                Norway       *
*    Jon K. Laerdahl          University of Oslo                Norway       *
*    Christoffer V. Larsen    University of Southern Denmark    Denmark      *
*    Patrick Norman           Linkoeping University             Sweden       *
*    Jeppe Olsen              Aarhus University                 Denmark      *
*    Markus Pernpointner      Vrije Universiteit Amsterdam      Netherlands  *
*    Jesper K. Pedersen       University of Southern Denmark    Denmark      *
*    Kenneth Ruud             University of Tromsoe             Norway       *
*    Pawel Salek              Stockholm Inst. of Technology     Sweden       *
*    Joost van Stralen        Vrije Universiteit Amsterdam      Netherlands  *
*    Joern Thyssen            University of Southern Denmark    Denmark      *
*    Olivier Visser           University of Groningen           Netherlands  *
*    Toke Winther             University of Southern Denmark    Denmark      *
*                                                                            *
*    This is an experimental code. The authors accept no responsibility      *
*    for the performance of the code or for the correctness of the results.  *
*                                                                            *
*    The code (in whole or part) is not to be reproduced for further         *
*    distribution without the written permission of the authors or           *
*    their representatives.                                                  *
*                                                                            *
*    If results obtained with this code are published, an                    *
*    appropriate citation would be:                                          *
*                                                                            *
*    "Dirac, a relativistic ab initio electronic structure program",         *
*    release DIRAC04.1 (2004),                                               *
*    written by H. J. Aa. Jensen, T. Saue, and L. Visscher                   *
*    with contributions from V. Bakken, E. Eliav, T. Enevoldsen, T. Fleig,   *
*    O. Fossgaard, T. Helgaker, J. K. Laerdahl, C. V. Larsen, P. Norman,     *
*    J. Olsen, M. Pernpointner, J. K. Pedersen, K. Ruud, P. Salek,           *
*    J. N. P. van Stralen, J. Thyssen, O. Visser, and T. Winther             *
*    (http://dirac.chem.sdu.dk).                                             *
*                                                                            *
*    (for a suitable BibTEX entry, see                                       *
*    <URL:"http://dirac.chem.sdu.dk/doc/reference.shtml">)                   *
*                                                                            *
******************************************************************************
 Version           : Dirac 4.1 (patchlevel 0) 
 Hostname          : radon.chem.vu.nl
 Operating system  : Darwin 8.8.0
 Machine           : Power Macintosh
 Run               : Sequential
 Last compilation  : 04:17:35 Oct 19 2006 (by visscher@radon.chem.vu.nl)
 FORTRAN compiler  : xlf -O3 -qstrict -qextname 
 C compiler        : /usr/bin/gcc-3.3 -O3 
 Size of COMMON= 192
 Basis set dir.    : /Users/visscher/sources/Dirac.autotrunk.serial/test/32.prjana/:/Users/visscher/sources/Dirac.autotrunk.serial/basis/:/Users/visscher/sources/Dirac.autotrunk.serial/basis_dalton/

 Date and time (IBM-AIX): Thu Oct 19 08:40:40 2006


 **************************************************************************
 ******************************* Fragment H *******************************
 **************************************************************************

 Jobs in this run:
   * Wave function
   * Analysis
HAMSCAN: Found keyword .LVCORR


 **************************************************************************
 ************************** General DIRAC set-up **************************
 **************************************************************************

 * The speed of light :        137.0359998
 * Direct evaluation of the following two-electron integrals:
   - LL-integrals
   - SL-integrals
   - SS-integrals
   - GT-integrals
 * Spherical transformation embedded in MO-transformation
   for large components
 * Transformation to scalar RKB basis embedded in
   MO-transformation for small components
 * Thresholds for linear dependence:
   Large components:   1.00D-06
   Small components:   1.00D-08
 * General print level   :   0


 *************************************************************************
 ****************** Output from HERMIT input processing ******************
 *************************************************************************



 *************************************************************************
 ****************** Output from READIN input processing ******************
 *************************************************************************



  Title Cards
  -----------

  Fragment H                                                              
  Base : cc-pVDZ                                                          

  Nuclear Gaussian exponent for atom of charge   1.000 :    2.1248239171D+09


                      SYMGRP:Point group information
                      ------------------------------

Point group: C1 

   * Character table

        |  E 
   -----+-----
    A   |   1

   * Direct product table

        | A  
   -----+-----
    A   | A  


                        **************************
                        *** Output from DBLGRP ***
                        **************************

   * One fermion irrep:   E1 
   * Quaternionic group. NZ = 4
   * Direct product decomposition:
          E1  x E1  : A   + A   + A   + A  


                             Spinor structure
                             ----------------


   * Fermion irrep no.: 1
      La  |  A  (1)  A  (1)  |
      Sa  |  A  (1)  A  (1)  |
      Lb  |  A  (1)  A  (1)  |
      Sb  |  A  (1)  A  (1)  |


                          Quaternion symmetries
                          ---------------------

    Rep  T(+)
    -----------------------------
    A    1  i  j  k


  Atoms and basis sets
  --------------------

  Number of atom types:     1
  Total number of atoms:    1

  label    atoms   charge   prim    cont     basis   
  ----------------------------------------------------------------------
  H           1       1       7       5      L  - [4s1p|2s1p]                                        
                             19      19      S  - [1s4p1d|1s4p1d]                                    
  ----------------------------------------------------------------------
                              7       5      L  - large components
                             19      19      S  - small components
  ----------------------------------------------------------------------
  total:      1       1      26      24

  Cartesian basis used.
  Threshold for integrals (to be written to file):  1.00D-15

  MNF.INP input file (for the AMFI stuff) is ready.


  References for the basis sets
  -----------------------------

  Atom type   1
   Elements                             References                                
   --------                             ----------                                
   H      : T.H. Dunning, Jr. J. Chem. Phys. 90, 1007 (1989).                     
   He     : D.E. Woon and T.H. Dunning, Jr. J. Chem. Phys. 100, 2975 (1994).      
   Li - Ne: T.H. Dunning, Jr. J. Chem. Phys. 90, 1007 (1989).                     
   Na - Mg: D.E. Woon and T.H. Dunning, Jr.  (to be published)                    
   Al - Ar: D.E. Woon and T.H. Dunning, Jr.  J. Chem. Phys. 98, 1358 (1993).      
   Ca     : J. Koput and K.A. Peterson, J. Phys. Chem. A, 106, 9595 (2002).       
   Ga - Kr: A.K. Wilson, D.E. Woon, K.A. Peterson, T.H. Dunning, Jr.,             
            J. Chem. Phys., 110, 7667 (1999)                                      


  Cartesian Coordinates
  ---------------------

  Total number of coordinates:  3


   1   H        x       .0000000000
   2            y       .0000000000
   3            z       .0000000000

  Nuclear repulsion energy :     .000000000000


                            GETLAB: AO-labels
                            -----------------

   * Large components:    4
     1  L H     s        2  L H     px       3  L H     py       4  L H     pz  
   * Small components:   10
     5  S H     s        6  S H     px       7  S H     py       8  S H     pz       9  S H     dxx     10  S H     dxy 
    11  S H     dxz     12  S H     dyy     13  S H     dyz     14  S H     dzz 


  Symmetry Orbitals
  -----------------

  Number of orbitals in each symmetry:           24
  Number of large orbitals in each symmetry:      5
  Number of small orbitals in each symmetry:     19

* Large component functions

  Symmetry  A  ( 1)

        2 functions:    H  s   
        1 functions:    H  px  
        1 functions:    H  py  
        1 functions:    H  pz  

* Small component functions

  Symmetry  A  ( 1)

        1 functions:    H  s   
        4 functions:    H  px  
        4 functions:    H  py  
        4 functions:    H  pz  
        1 functions:    H  dxx 
        1 functions:    H  dxy 
        1 functions:    H  dxz 
        1 functions:    H  dyy 
        1 functions:    H  dyz 
        1 functions:    H  dzz 


 ***************************************************************************
 *************************** Hamiltonian defined ***************************
 ***************************************************************************

 * Print level :    0
 * Dirac-Coulomb Hamiltonian
 * SS integrals neglected:
   Interatomic Coulombic SS-contributions modelled by
   classical repulsion of small component atomic charges
   using tabulated charges.
 * Default integral flags passed to all modules
   - LL-integrals:     1
   - LS-integrals:     1
   - SS-integrals:     0
   - GT-integrals:     0
 * Basis set:
   - contracted large component basis set
   - uncontracted small component basis set


 Information about the restricted kinetic balance scheme:
 * Default RKB projection:
   1: Pre-projection in scalar basis
   2: Removal of unphysical solutions (via diagonalization of free particle Hamiltonian)


 *********************************************************************************
 *************************** Memory required by HERMIT ***************************
 *********************************************************************************

===========================================================================
 Maximum memory load for two-electron integral processing:
===========================================================================
 * LL-integrals:
   1(H    Ls)   1(H    Ls)   1(H    Ls)   1(H    Ls)                2638
 * SL-integrals:
   4(H    Sp)   4(H    Sp)   1(H    Ls)   1(H    Ls)                9467


 **************************************************************************
 ************************** Wave function module **************************
 **************************************************************************

 Jobs in this run (in execution order):
 * Hartree-Fock calculation
===========================================================================
 DHFINP: Set-up for Hartree-Fock calculation:
===========================================================================
 * Number of fermion irreps: 1
 * Open shell SCF calculation using Average-of-Configuration

 * Shell specifications:

                                        Orbitals
                        #electrons  irrep 1  irrep 2     f        a      alpha
                        ----------  -------  -------  -------  -------  -------
   Closed shell               0          0      N/A    1.0000   1.0000    .0000
   Open shell no. 1           1          1      N/A     .5000    .0000   2.0000
   ----------------------------------------------------------------------------
   Total                      1          1

   f is the fraction occupation; a and alpha open shell coupling coefficients.

 * Bare nucleus screening correction used for start guess
 * General print level   :   0
 ***** TRIAL FUNCTION *****
 * Trial vectors read from file DFCOEF
 ***** CONVERGENCE CRITERIA *****
 * Convergence on norm of error vector (gradient).
   Desired convergence:1.000D-07
   Allowed convergence:1.000D-06

 ***** CONVERGENCE CONTROL *****
 * Fock matrix constructed using differential density matrix
    with optimal parameter.
 * DIIS (in MO basis)
 * DIIS will be activated when convergence reaches : 1.00D+20
   - Maximum size of B-matrix:   10
 * Damping of Fock matrix when DIIS is not activated. 
   Weight of old matrix    :  .250
 * Maximum number of SCF iterations  :   50
 * No quadratic convergent Hartree-Fock
 * Contributions from 2-electron integrals to Fock matrix:
   LL-integrals.
   SL-integrals from iteration    1
 ***** OUTPUT CONTROL *****
 * Only electron eigenvalues written out.


 ***************************************************************************
 ***************************** Analysis module *****************************
 ***************************************************************************

 Jobs in this run:
 * Mulliken population analysis
===========================================================================
 POPINP: Mulliken population analysis
===========================================================================
 * Gross populations
 * Label definitions based on SO-labels
 * Number of spinors analyzed:
    - Orbitals in fermion ircop E1  :   1                                                                    
 * Print level:    0


 ********************************************************************************
 *************************** Input consistency checks ***************************
 ********************************************************************************



 *************************************************************************
 ************************ End of input processing ************************
 *************************************************************************



                        Generating Lowdin matrix:
                        -------------------------

   L   A     * Deleted:          0(Proj:          0, Lindep:          0)
   S   A     * Deleted:          1(Proj:          1, Lindep:          0)


                            Output from MODHAM
                            ------------------

* Applied strict kinetic balance !


  **********************************************************************
  ************************* Orbital dimensions *************************
  **********************************************************************

No. of electronic orbitals (NESH):     5
No. of positronic orbitals (NPSH):     5
Total no. of orbitals      (NORB):    10


 ****************************************************************************
 ************************* Hartree-Fock calculation *************************
 ****************************************************************************


*** INFO *** No trial vectors found. Using bare nucleus approximation for initial trial vectors.
             Improved by an estimate of the electronic screening (Slater's rules).


########## START ITERATION NO.   1 ##########   Thu Oct 19 08:40:40 2006

It.   1    -.1730868868079      1.00D+20  0.00D+00  0.00D+00                .01000000s   Bare nuclei    Thu Oct 19

########## START ITERATION NO.   2##########   Thu Oct 19 08:40:40 2006


* GETGAB: label "GABAO1XX" not found; calling GABGEN.
SCR        scr.thr.    Step1    Step2  Coulomb  Exchange    CPU-time
SOfock:LL  1.00D-12   47.06%   33.69%   24.13%   32.36%    .00000000s
SOfock:SL  1.00D-12   26.92%   51.87%   23.75%   27.66%    .00000000s
>>> Total wall time:  .00000000s
>>> Total CPU-time :  .01000000s

########## END ITERATION NO.   2 ##########   Thu Oct 19 08:40:40 2006

It.   2    -.4980519996414      3.25D-01  3.56D-01  2.90D-02                .01000000s   LL SL          Thu Oct 19

########## START ITERATION NO.   3##########   Thu Oct 19 08:40:40 2006

    3 *** Differential density matrix. DVOVLP( 1) = 1.1071
SCR        scr.thr.    Step1    Step2  Coulomb  Exchange    CPU-time
SOfock:LL  1.00D-12   47.06%   33.69%   24.13%   32.36%    .00000000s
SOfock:SL  1.00D-12   26.92%   51.87%   23.75%   27.66%    .00000000s
>>> Total wall time:  .00000000s
>>> Total CPU-time :  .01000000s

########## END ITERATION NO.   3 ##########   Thu Oct 19 08:40:40 2006

It.   3    -.4990865893398      1.03D-03  1.30D-02  1.16D-02   DIIS   2     .01000000s   LL SL          Thu Oct 19

########## START ITERATION NO.   4##########   Thu Oct 19 08:40:40 2006

    4 *** Differential density matrix. DVOVLP( 1) = 1.0728
SCR        scr.thr.    Step1    Step2  Coulomb  Exchange    CPU-time
SOfock:LL  1.00D-12   47.06%   33.69%   24.13%   32.36%    .00000000s
SOfock:SL  1.00D-12   26.92%   51.87%   23.75%   27.66%    .00000000s
>>> Total wall time:  .00000000s
>>> Total CPU-time :  .01000000s

########## END ITERATION NO.   4 ##########   Thu Oct 19 08:40:40 2006

It.   4    -.4992847534046      1.98D-04  8.63D-03  1.01D-05   DIIS   3     .01000000s   LL SL          Thu Oct 19

########## START ITERATION NO.   5##########   Thu Oct 19 08:40:40 2006

    5 *** Differential density matrix. DVOVLP( 1) =  .9999
SCR        scr.thr.    Step1    Step2  Coulomb  Exchange    CPU-time
SOfock:LL  1.00D-12   47.06%   33.69%   24.13%   32.36%    .00000000s
SOfock:SL  1.00D-12   26.92%   53.88%   23.38%   25.75%    .00000000s
>>> Total wall time:  .00000000s
>>> Total CPU-time :  .00000000s

########## END ITERATION NO.   5 ##########   Thu Oct 19 08:40:40 2006

It.   5    -.4992847535525      1.48D-10 -7.46D-06  6.53D-08   DIIS   3     .00000000s   LL SL          Thu Oct 19


                               SCF - CYCLE
                               -----------

* Convergence on norm of error vector (gradient).
  Desired convergence:1.000D-07
  Allowed convergence:1.000D-06

* ERGVAL - convergence in total energy
* FCKVAL - convergence in maximum change in total Fock matrix
* EVCVAL - convergence in error vector (gradient)
--------------------------------------------------------------------------------------------------------------------------------
           Energy               ERGVAL    FCKVAL    EVCVAL      Conv.acc    CPU          Integrals   Time stamp
--------------------------------------------------------------------------------------------------------------------------------
It.   1    -.1730868868079      1.00D+20  0.00D+00  0.00D+00                .01000000s   Bare nuclei    Thu Oct 19
It.   2    -.4980519996414      3.25D-01  3.56D-01  2.90D-02                .01000000s   LL SL          Thu Oct 19
It.   3    -.4990865893398      1.03D-03  1.30D-02  1.16D-02   DIIS   2     .01000000s   LL SL          Thu Oct 19
It.   4    -.4992847534046      1.98D-04  8.63D-03  1.01D-05   DIIS   3     .01000000s   LL SL          Thu Oct 19
It.   5    -.4992847535525      1.48D-10 -7.46D-06  6.53D-08   DIIS   3     .00000000s   LL SL          Thu Oct 19
--------------------------------------------------------------------------------------------------------------------------------
* Convergence after    5 iterations.
* Average elapsed time per iteration: 
      No 2-ints    :     .00000000s
      LL SL        :     .00000000s


                               TOTAL ENERGY
                               ------------

   Electronic energy                        :    -.49928475355246349

   Other contributions to the total energy
   Nuclear repulsion energy                 :     .00000000000000000
   SS Coulombic correction                  :     .00000000000000000

   Sum of all contributions to the energy
   Total energy                             :    -.49928475355246349


                               Eigenvalues
                               -----------


* Fermion symmetry E1 
  * Open shell #1, f =  .5000
 -.499282719848589  ( 2)
  * Virtual eigenvalues, f =  .0000
  .636502797549630  ( 2)    1.508219213010438  ( 2)    1.508255990010211  ( 4)
* HOMO - LUMO gap:

    E(LUMO) :      .63650280 au (symmetry E1 )
  - E(HOMO) :     -.49928272 au (symmetry E1 )
  ------------------------------------------
    gap     :     1.13578552 au



 **************************************************************************
 ********************** Mulliken population analysis **********************
 **************************************************************************



                            Fermion ircop E1 
                            -----------------


* Electronic eigenvalue nr.  1:  -.4992827198486       (Occupation : f =  .5000)
================================================================================

* Gross populations greater than  .00010

Gross     Total   |    L A  H  s   
--------------------------------------
 alpha    1.0000  |      1.0000
 beta      .0000  |       .0000


*** Total gross population ***

Gross      Total    |    L A  H  s   
--------------------------------------
 total     1.00000  |       .99999

 Date and time (IBM-AIX): Thu Oct 19 08:40:40 2006
*****************************************************
>>>> Total CPU  time used in DIRAC:  0.07000000s
>>>> Total WALL time used in DIRAC:  1.00000000s
********** E N D   of   D I R A C  output  **********
